ts/src/hyperliquid.ts

```diff
@@ -361,6 +361,52 @@ export default class hyperliquid extends Exchange {
         return this.parseMarkets (result);
     }
 
+    /**
+     * @method
+     * @name calculatePricePrecision
+     * @description Helper function to calculate the Hyperliquid DECIMAL_PLACES price precision
+     * @param {float} price the price to use in the calculation
+     * @param {int} amountPrecision the amountPrecision to use in the calculation
+     * @param {int} maxDecimals the maxDecimals to use in the calculation
+     * @returns {int} The calculated price precision
+     */
+    calculatePricePrecision (price: number, amountPrecision: number, maxDecimals: number) {
+        let pricePrecision = 0;
+        // We might get 2 dots, doesn't matter
+        const priceStr = this.numberToString (price) + '.';
```

Comment:
@gaardiolor why do we need this this + '.' here?

Reply:
Hmm I thought it was needed for cases where price was an int. But the code that has `priceStr.split ('.')[1];` will not be hit if it is an int, and `priceStr.split('.')[0]` will always work (in python at least). I think I can delete it

Reply:
yeah and if you're accessing the first element, `100`.split('.') or '100.'.split('.') will produce the same result

Reply:
ok fixed